src/tusb_option.h

```diff
@@ -61,6 +61,8 @@
 #define OPT_MCU_SAME5X            203 ///< MicroChip SAM E5x
 #define OPT_MCU_SAMD11            204 ///< MicroChip SAMD11
 #define OPT_MCU_SAML22            205 ///< MicroChip SAML22
+#define OPT_MCU_SAM7X             206 ///< MicroChip SAME70, S70, V70, V71 family
```

Comment:
there is actually an SAM7X which is different mcu, https://www.microchip.com/developmenttools/ProductDetails/PartNO/AT91SAM7X-EK

Maybe call it `SAMX7X`, I am not good for naming, any other suggestion is welcome.

Reply:
Yes `SAMX7X` works... I can't figurer out something better neither 😅

Reply:
ups my bad, had no knowledge of sam7x being a thing, and didnt think to search.

Reply:
not a problem at all. I like your naming more, but unfortunately, avoiding confusion has more priority :)